### DISCUSSION TITLE
How to query an item that has at least one relation?

### DISCUSSION THREAD

**Discussion by Akumzy** [prisma/prisma#8421] [Q&A] [answered]
Hi, I want to return all products that have reviews and I can't seem to figure that out since I just want to check the length of the reviews

```ts
 const products = await prisma.product.findMany({
      where: {  reviews: {  } },
  })

```

**Comment by ryands17** [comment, marked as answer]
You can use [some](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#filter-on-presence-of-related-records) in this case:

```ts
 const products = await prisma.product.findMany({
      where: {  reviews: { some: {} } },
  })
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
